### PR TITLE
Initialise bookstore project with components

### PR DESCRIPTION
**Pull request by stephenkati**
- Created **React user interface** for **bookstore**
- Installed **React router**
- **Setup client-side routing** using **`React router v6`**
- Created components inside **`/components` folder**:

1. Books
2. Categories

### CODE REVIEW

File: src/components/books.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <span>Jane Austen</span>
+      </li>
+    </ul>
+    <form>
+      <input type="text" placeholder="Book title..." />
+      <input type="text" placeholder="Book Author" />
+      <input type="submit" />
+    </form>
+  </div>
+);
```
Comment (Ibizugbe) [inline src/components/books.js]: Nice Job done so far, You have created the `books` page and the `categories` page as components👍. But for the books page, I suggest you take note of the following and make the necessary change👍.

- [ ] Just as stated in the requirements for this milestone you are to create a component that;
- Contains individual book state, as well as a button to delete the book (more like a component to display a single book)😉.
- Contains a list to render individual books (more like components that shows all your books)😉.
- Contains a form to create new books (a form that lets the user add a new book)😉.
- Contains navigation to link to the 2 routes you created (This you have already implemented👍).

components

kindly create a component for each of these items and import them to your `books` page where they will be used😉. Breaking your page into components helps for reusability and separation of concerns which makes it easy to make update to your app👍.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @stephenkati 👋👋👋, 

tryagain


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are properly setup and they are passing👍.
- [x] Project is initialized using `create react app`👍.
- [x] `React-router-dom` have been properly installed👍.
- [x] The app contains two page components for the `books` and `categories`👍.
- [x] Gitflow have been rightly used👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [OPTIONAL] Your commit messages are meaningful and descriptive👍. but for supsequent times, I recommend you capitalize them to make it more professional👍.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/books.js (hunk 1 above)
Comment (stephenkati) [reply src/components/books.js]: Alright on it right away

**Review by sumairq — APPROVED** [review]
### Hi @stephenkati , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**